matrix-meetings-bot/src/service/RoomMessageService.ts

```diff
@@ -325,6 +351,18 @@ export class RoomMessageService {
       notification += this.createDescriptionLine(lng, oldDescription, true);
     }
 
+    if (meetingChanges.calendarChanged) {
+      const newRruleText = isRecurringCalendarSourceEntry(newMeeting.calendar)
+        ? formatRRuleText(newMeeting.calendar[0].rrule, i18next.t, lng)
+        : '';
```

Comment:
The widget shows the following in case there is no repetition:
<img width="700" alt="image" src="https://github.com/nordeck/matrix-meetings/assets/720821/22aa1602-4d69-4fd5-9c13-b02743d18c6c">

<img width="707" alt="image" src="https://github.com/nordeck/matrix-meetings/assets/720821/fb6c5f34-3484-439a-aee3-f48000ac5e16">

This could be a good alternative instead of showing an empty string:

<img width="400" alt="image" src="https://github.com/nordeck/matrix-meetings/assets/720821/ea1ef1bc-c371-4612-adb2-105ca972deee">
vs
<img width="248" alt="image" src="https://github.com/nordeck/matrix-meetings/assets/720821/e2afc356-b22f-4698-b4df-8d38c2cba6f4">

Reply:
good point, thanks!